docs/intro.md

```diff
@@ -1 +1,26 @@
-**Telescope ALBA** documentation.
+- Assume that, a prover, who has a large collection of data, wants to convince a verifier that their set contains at least a minimum number of items that satisfy a specific condition, known as a predicate, even if the prover only shares a portion of it.
+- The Approximate Lower Bound Argument (ALBA) protocol is a new cryptographic primitive that solves this problem efficiently.
+- The protocol makes use of a small gap between the size of what the prover actually has and the threshold the verifier knows they have.
+- . This gap makes ALBA very efficient, as it reduces the data the prover needs to share, enabling compact and efficient proofs.
+- ALBA also supports weighted items, where each item has an assigned importance or weight. This allows the prover to demonstrate that the total weight of their set meets a required threshold, rather than simply the count of items.
+- ALBA supports scalable and efficient proof mechanisms in decentralized systems, i.e., voting schemes.
+
+## Overview
+- **The protocol**
+    - The protocol addresses the problem of succinctly proving knowledge of a large set of verifiable evidence.
+    - The prover convinces the verifier by revealing only a subset of this evidence, thus achieving efficiency in both time and communication.
+    - Given a large set $S_p$ that satisfies a predicate $R$ such that $|S_p| \geq n_p$, the prover wants to convince the verifier that the set contains at least $n_f$ elements, where $n_f$ is a threshold strictly smaller than $n_p$.
+        - This creates an _approximate_ lower bound, as the verifier is convinced that $S_p$ meets or exceeds a threshold, though the actual size might be greater.
+    - ALBA achieves efficiency by making use of the gap between the provable lower bound and the actual size, enabling rapid verification without compromising security. The greater the gap, the smaller the proof, and the faster its generation.
+- **Historical context**
+    - The concept builds on classic approaches in proof systems, where similar challenges in communication complexity were addressed by using probabilistic techniques or interactive protocols.
+    - Previous methods were largely theoretical and less efficient for practical, large-scale applications.
+- **Design goals**
+    - ALBA aims to minimize both proof size and computational load, making it feasible for high-throughput settings like blockchain networks.
+    - The protocol is versatile and adaptable, supporting non-interactive proofs in both the random oracle and common reference string model as well as allowing multi-prover scenario in decentralized environments.
```

Comment:
I think this could also be more concise. We could say "ALBA works in various settings: unweighted/weighted and centralized/decentralized. It also works in two different models: random oracle and common reference string.". I'm not sure "versatile and adaptable" add anything. "Non-interactive proofs" should be stated somewhere at the beginning of this file.